Model/CRUD/UpdateCronTask.php

```diff
@@ -38,15 +38,15 @@ public function __construct(EntityManager $entityManager, CommandValidator $comm
     /**
      * Updates an existing cron task
      *
-     * @param TblCronTask $cronTask    CronTask to update
-     * @param string      $name        Name for the cron task
-     * @param string      $description Description of the cron
-     * @param string      $command     Command string (i.e.: "cache:clear")
-     * @param int         $interval    Number of seconds between command executions
+     * @param TblCronTask $cronTask       CronTask to update
+     * @param string      $name           Name for the cron task
+     * @param string      $description    Description of the cron
+     * @param string      $command        Command string (i.e.: "cache:clear")
+     * @param string      $cronExpression Cron expression (https://en.wikipedia.org/wiki/Cron)
```

Comment:
For the link => http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlinelink.pkg.html